src/constants.py

```diff
@@ -0,0 +1,7 @@
+"""
+This module contains constants used throughout the application.
+"""
+
```

Comment:
Hi @MHryhoriev !
Can we add the user directory here as well?

Examples:
from pathlib import Path
path = Path.home().joinpath( 'Documents', 'mysql_access' )
or
path_string = '~\Documents\mysql_access'
path = Path(path_string).expanduser()

Reply:
You mean dynamically and cross-platform create a path to our data files?
Something like that:
CONTACT_DATA_FILE_PATH = Path.home().joinpath('data', 'contacts_data.json')
NOTE_DATA_FILE_PATH = Path.home().joinpath('data', 'notes_data.json')

Or do you mean some other user directory?